src/commands/config.ts

```diff
@@ -7,10 +7,13 @@ import { intro, outro } from '@clack/prompts';
 import chalk from 'chalk';
 import { COMMANDS } from '../CommandsEnum';
 
+import * as dotenv from 'dotenv'
+dotenv.config()
+
 export enum CONFIG_KEYS {
```

Comment:
@di-sukharev A new setting was added in this PR: https://github.com/di-sukharev/opencommit/pull/31. 
So it will be necessary to update the branch to adjust it